assets/src/block-editor/index.js

```diff
@@ -16,20 +16,23 @@ import './store';
 
 const { ampLatestStoriesBlockData } = window;
 
-addFilter( 'blocks.registerBlockType', 'ampEditorBlocks/addAttributes', addAMPAttributes );
-addFilter( 'blocks.getSaveElement', 'ampEditorBlocks/filterSave', filterBlocksSave );
-addFilter( 'editor.BlockEdit', 'ampEditorBlocks/filterEdit', filterBlocksEdit, 20 );
-addFilter( 'blocks.getSaveContent.extraProps', 'ampEditorBlocks/addExtraAttributes', addAMPExtraProps );
-addFilter( 'editor.PostFeaturedImage', 'ampEditorBlocks/withFeaturedImageNotice', withFeaturedImageNotice );
-addFilter( 'editor.MediaUpload', 'ampEditorBlocks/addCroppedFeaturedImage', ( InitialMediaUpload ) => withCroppedFeaturedImage( InitialMediaUpload, getMinimumFeaturedImageDimensions() ) );
+// Add filters if AMP for Website experience is enabled.
+if ( select( 'amp/block-editor' ).isWebsiteEnabled() ) {
```

Comment:
@swissspidy How do these changes in 05dda18f7b2914c259c8c2c2fa9e525e36822ab9 look?

Reply:
Good at first glance 👍 Just made some minor edits.